modules/packages/UnorderedCopy.chpl

```diff
@@ -93,6 +93,16 @@ module UnorderedCopy {
     }
   }
 
+  /*
+     Fence any pending unordered copies issued by the current task.
+   */
+  inline proc unorderedCopyTaskFence(): void {
+    if CHPL_COMM == 'ugni' {
```

Comment:
This condition is different from the one in `UnorderedATomics.chpl` (that one will eventually let comm=gasnet through to do a fence; this one never will).  That difference seems subtle -- should we comment as to the reason?

Reply:
Soon (in the next sprint or 2) the API will be extended to all comm layers, so I don't mind leaving this undocumented for now.